scripts/index.js

```diff
@@ -0,0 +1,183 @@
+function setQuery(evt) {
+    if (evt.keyCode == 13) {
+        getResults(searchbox.value);
+        console.log(searchbox.value);
+    }
+}
+
+//when keypress event initialized it will run a fetch request. Get API,attach weather info based on our query that've entered to searchbox value
+function getResults (query) {
+    fetch(`${api.baseurl}weather?q=${query}&units=metric&APPID=${api.key}`)
+    .then(weather => {
+        console.log(weather);
+        return weather.json();
+    }).then(setGlobalTempValue).then(displayResults);
+}
+
+function displayResults (weather) {
+    console.log(weather);
+    const city = document.querySelector('.location .city');
+    city.innerText = `${weather.name}, ${weather.sys.country}`;
+
+    const now = new Date();
+    const date = document.querySelector('.location .date');
+    date.innerText = dateBuilder(now);
+    //temp
+    const temp = document.querySelector('.current #temp');
+    temp.innerHTML = `${Math.round(weather.main.temp)}<span>°c</span>`;
+
+    const weather_elem = document.querySelector('.current .weather');
+    weather_elem.innerText = weather.weather[0].main;
+
+    const hilow = document.querySelector('.hi-low');
+    hilow.innerText = `${Math.round(weather.main.temp_min)}°c / ${Math.round(weather.main.temp_max)}°c`;
+}
+
+//set data from openWeather's API - dynamic temperature value to my global variable
+function setGlobalTempValue(weather) {
+    globalTempValue = Math.round(weather.main.temp);
+    return weather
+}
+
+function dateBuilder (date_attr) {
+    let months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+    let days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+
+    const day = days[date_attr.getDay()];
+    const date = date_attr.getDate();
+    const month = months[date_attr.getMonth()];
+    const year = date_attr.getFullYear();
+
+    return `${day} ${date} ${month} ${year}`;
+}
+
+function updateCityName() {
+    const inputName = document.getElementById("inputSearchBox").value;
+    const headerCityName = document.getElementById("headerCity");
+    headerCityName.textContent = inputName;
+};
+
+function resetCityName() {
+    const resetSearchBox = document.getElementById("inputSearchBox");
+    resetSearchBox.value = "Search for a city";
+}
+
+function setSky() {
+    const getSky = document.getElementById("skyDropdown").value;
+    const displaySky = document.getElementById("sky");
+
+    let sky = "";
+    let skyColor = "";
+    if (getSky === "Cloudy") {
+        sky = "☁️☁️🌤☁️☁️";
+        skyColor = "cloudy";
+    } else if (getSky === "Sunny") {
+        sky = "☁️ ☁️ ☀️ ☁️  ☁️";
+        skyColor = "sunny";
+    } else if (getSky === "Rainy") {
+        sky = "🌧🌈💧🌧🌧";
+        skyColor = "rainy";
+    } else if (getSky === "Snowy") {
+        sky = "🌨❄️🌨🌨";
+        skyColor = "snowy";
+    }
```

Comment:
This is a great place to use a Javascript switch statement (https://www.w3schools.com/js/js_switch.asp).